### PR TITLE
Fix tests for Rails 5.2

### LINKED ISSUE #14: Build failures on master
Builds are broken for Rails 5.2 due to a missing method `ActiveRecord::Migrator.migrate`. Similar issue in other project: https://github.com/thiagopradi/octopus/issues/477.

### PR DESCRIPTION
**Pull request by skalee**
Rails 5.2 introduces a concept of migration contexts.  More details on that can be found here:

- https://github.com/rails/rails/commit/a2827ec9811b5
- https://kirshatrov.com/2018/04/01/async-migrations/

Fixes #14.

### CODE REVIEW

**Comment by skalee** [comment]
One issue has been fixed, but it is not Rails-5.2-ready yet. The problem possibly comes from some changes in ActiveRecord's private APIs. That said I cannot say that I fully understand its nature already…

File: activeuuid.gemspec [hunk 1]
```diff
@@ -21,6 +21,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency "database_cleaner"
   s.add_development_dependency "fabrication"
   s.add_development_dependency "forgery"
+  s.add_development_dependency "pry"
```
Comment (jlw) [inline activeuuid.gemspec:24]: FYI, pry-byebug adds both pry and byebug, so pry doesn't need to be declared as a separate dependency.

File: gemfiles/Rails-5_2.gemfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+source "http://rubygems.org"
```
Comment (jlw) [inline gemfiles/Rails-5_2.gemfile:1]: This source should use HTTPS.

File: activeuuid.gemspec (hunk 1 above)
Comment (ronaldtse) [reply activeuuid.gemspec:24]: Agree, thanks @jlw !

File: gemfiles/Rails-5_2.gemfile (hunk 2 above)
Comment (ronaldtse) [reply gemfiles/Rails-5_2.gemfile:1]: Absolutely. Thanks @jlw !

**Comment by ronaldtse** [comment]
@skalee please feel free to merge when ready. Thanks!

**Comment by skalee** [comment]
I have it done better on some side branch. In fact, that other branch is quite a rewrite. Migration issues are not the only ones.

**Comment by ronaldtse** [comment]
Sounds good — then is that other branch ready soon? If so we can close this current one to prevent confusion.

**Comment by skalee** [comment]
Closing in favour of #27.

**Pull request closed without merging**